app/components/Farming/FarmingCard.tsx

```diff
@@ -1,80 +1,146 @@
+        <div className={`relative rounded-3xl p-[1px] max-w-md min-w-[350px] ${color === Color.BLUE ? "bg-farming-border-blue" : ""} ${color === Color.GREEN ? "bg-farming-border-green" : ""} ${color === Color.BROWN ? "bg-farming-border-orange" : ""} hover:brightness-[108%]`}>
             <NavLink to={`/farming/${project.slug}`} className="w-full">
                 <div className="w-full bg-neutral-800 rounded-t-3xl">
-                    <div className={`relative rounded-t-3xl p-6 ${type === 'blue' ? "bg-farming-header-blue" : ""} ${type === 'green' ? "bg-farming-header-green" : ""} ${type === 'orange' ? "bg-farming-header-orange" : ""}`}>
+                    <div className={`relative rounded-t-3xl p-6 ${color === Color.BLUE ? "bg-farming-header-blue" : ""} ${color === Color.GREEN ? "bg-farming-header-green" : ""} ${color === Color.BROWN ? "bg-farming-header-orange" : ""}`}>
                         <div className="grid grid-cols-2">
                             <div className="text-left">
                                 <div className="font-inter text-neutral-100">My stake</div>
-                                <div className="font-inter text-neutral-300 mt-1">$0</div>
+                                <div className="font-inter text-neutral-300 mt-1"><span className='mr-[2px]'>$</span>{myStake}</div>
                             </div>
                             <div className="text-right">
                                 <div className="font-inter text-neutral-100">Farming APR</div>
-                                <div className="font-inter text-neutral-300 mt-1">23.6%</div>
+                                <div className="font-inter text-neutral-300 mt-1">{apr}<span className='ml-[2px]'>%</span></div>
                             </div>
                         </div>
                     </div>
-                    <div className={`w-full h-[1px] ${type === 'blue' ? "bg-farming-separator-blue" : ""} ${type === 'green' ? "bg-farming-separator-green" : ""} ${type === 'orange' ? "bg-farming-separator-orange" : ""}`}></div>
+                    <div className={`w-full h-[1px] ${color === Color.BLUE ? "bg-farming-separator-blue" : ""} ${color === Color.GREEN ? "bg-farming-separator-green" : ""} ${color === Color.BROWN ? "bg-farming-separator-orange" : ""}`}></div>
                     <div className="relative text-center p-4 bg-farming-card-bg">
-                        <img src={IPFS_GATEWAY + project.imageIpfs} alt={`${project.slug} NFT card`} className="w-[66px] rounded-full absolute top-[-33px] left-[calc(50%_-_33px)] border border-neutral-50" />
+                        <img src={IPFS_GATEWAY + ipfsUrl(project.Uri.data.image)} alt={`${project.slug} NFT card`} className="w-[66px] rounded-full absolute top-[-33px] left-[calc(50%_-_33px)] border border-neutral-50" />
```

Comment:
can we imagine something like ${printColorClass(color)} ? to many repititions there

Reply:
Done. Needed a little trick because tailwind doesn't generate css for dynamically build classes. I used the safelist in tailwind config to make it work as recommended in there documentation.